### DISCUSSION TITLE
Is there any reason this wouldn't work on a raspberry pi4 w/supported touch screen?

### DISCUSSION THREAD

**Discussion by JustTheDazz** [Guilouz/Klipper-Flsun-Speeder-Pad#51] [General]
I was thinking about doing this with the above setup, would there be any reason why it wouldn't work?

**Comment by Guilouz** [comment]
Image file is not for Raspberry Pi but only for Speeder Pad

**Reply by JustTheDazz** [reply]
Ok, thank you

**Reply by yukdumboobumm** [reply]
I just did this. Use the [Raspberry Pi Imager](https://www.raspberrypi.com/software/) to image an Sd Card with the official 32bit Mainsail OS and then follow all of @Guilouz  instructions  starting from [Install official builds](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad#install-official-builds-1-instance)

pi4 is **much** faster than speeder pad.

**Comment by Freddyb888** [comment]
And can I use the btt pad 7 with cb1 emmc by following your great install for the speeder pad? Thanks

**Reply by Guilouz** [reply]
CB1 does not use the same OS but you can use my configurations files with CB1.
